content/en/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -92,35 +93,82 @@ Starting with [version 2.17.0][3], you can add view names and assign them to a d
    {{% tab "NPM" %}}
    ```javascript
    datadogRum.startView({
-     name: 'checkout',
-     service: 'purchase',
-     version: '1.2.3'
+    name: 'checkout',
+    service: 'purchase',
+    version: '1.2.3',
+    context: {
+      team: 'payment'
+    },
    })
    ```
 
    {{% /tab %}}
    {{% tab "CDN async" %}}
    ```javascript
    window.DD_RUM.onReady(function() {
-       window.DD_RUM.startView({
-         name: 'checkout',
-         service: 'purchase',
-         version: '1.2.3'
-       })
+      window.DD_RUM.startView({
+        name: 'checkout',
+        service: 'purchase',
+        version: '1.2.3',
+        context: {
+          team: 'payment'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** `‏team` is specific a concept in Datadog. I think we should another example: Maybe
```
context: {
          payment: 'Done'
```

Reply:
Changed it to this ⬆️